### PR TITLE
Add clean button to SerialPlot

### PR DESCRIPTION
**Pull request by chromhelm**

### CODE REVIEW

**Comment by facchinm** [comment]
Hi @chromhelm ,
thanks for the contribution! Could you cleanup it a bit?
Since you implement three functionalities (clear button, labels and configurable buffer) it would be better to have three different commits, hence removing f8db03e from the history and splitting c76a8f9 into two (or more) commits.

File: app/src/processing/app/SerialPlotter.java [hunk 1]
```diff
@@ -287,19 +316,81 @@ public void message(final String s) {
...
       int validParts = 0;
+      int validLabels = 0;
       for(int i = 0; i < parts.length; ++i) {
-        try {          
-          double value = Double.valueOf(parts[i]);
+        Double value = null;
+        String label = null;
+        
+        // all commands start with #
+        if(parts[i].startsWith("#") && parts[i].length() > 1) {
```
Comment (facchinm) [inline app/src/processing/app/SerialPlotter.java]: I'm not a big fan of hardcoded commands but I appreciate the idea. 
Probably a standalone commit (like "add CLEAR and SET_SIZE runtime commands to Serial Plotter") could do the trick and make discussion easier

**Comment by chromhelm** [comment]
Hi
I made a new pull request for label implementation #7453

File: app/src/processing/app/SerialPlotter.java (hunk 1 above)
Comment (chromhelm) [reply app/src/processing/app/SerialPlotter.java]: Done in pull request #7461

**Comment by madsdyd** [comment]
This PR has a label of "IDE 1.9.x Beta". What does this mean? 

I think the relationship between #7452, #7454 and #7461 is confusing. Does anyone know what the status is?

**Pull request closed without merging**